packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_loan_place_auction_bid.test.ts

```diff
@@ -269,8 +299,8 @@ async function setup (): Promise<void> {
   expect(vaultAfter.liquidationHeight).toStrictEqual(expect.any(Number))
   expect(vaultAfter.liquidationPenalty).toStrictEqual(5)
   expect(vaultAfter.batches).toStrictEqual([
-    { index: 0, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00399539@TSLA' },
-    { index: 1, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00399539@TSLA' }
+    { index: 0, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00342462@TSLA' },
```

Comment:
looking at the interest changes when we update the test case, this is mainly due to the following facts.
1. price activation is mainly according to a fixed price interval. i.e for `regtest`, every 6 blocks a price update block comes.
2. so if we add more code that generates extra blocks, then the code flow w.r.t. `fixed price interval` might change. As a result, the number of blocks to wait for events like `liquidation` might differ. Hence the interest numbers differ.

This exposes a deeper concern about test maintainability. Again manual calculation and verification takes a considerable amount of review time. 
I could not think of a perfect solution. We could try to update the tests in a way that the `take loan` action happens after a sync with the `fixed price interval`. But again this does not address the whole problem. we could always do a calculation based solution with some extra effort instead of hardcoding the values. But then it affects the test readability.

Reply:
From another perspective, tests are not expected to withstand when you change the test content. So that's something have to bear. 

So finding the middle ground between these two might be a better solution.

Reply:
> tests are not expected to withstand when you change the test content

Definitely agree, just say we need to minimize test content changes esp the code change is solely for refactoring purpose.

Reply:
2 ways of assertion
current - clear and readability but harder to maintain while minor code change
```ts
{ index: 0, collaterals: ['5000.00000000@DFI', '0.50000000@BTC'], loan: '500.00342462@TSLA' },
```
or 
hmm.. maybe less clear but easy to maintain
```ts
expect(loan).toStrictEqual(loamAmt + interestPerBlock * numBlk) // 500.00342462@TSLA'
```